backend/manifest.yml

```diff
@@ -1,6 +1,5 @@
 ---
 applications:
-- name: backend
-  health-check-type: http
-  routes:
-  - route: backend.apps.internal
+- name: paas-frontend-backend-example-backend
```

Comment:
this name is _really_ long given it will be typed a lot by a user at the command line in a user research session. Also, having 'frontend-backend' in the name is confusing. Same goes for the frontend app.

Suggestion: name the apps something simple like gac-{frontend,backend} (Government Art Collection).

Reply:
Very good point - I've renamed to `gac-{front,back}end` as suggested 👍